### ISSUE TITLE
Gateway address public key not being generated

### ISSUE THREAD

**Issue by roosmaa** [jansenmarc/WavesGatewayFramework#20]
I'm guessing PyWaves library changed it's behavior slightly at some point, and that before the following used to work and properly generate the publicKey from the provided privateKey:

https://github.com/jansenmarc/WavesGatewayFramework/blob/b3674a3a5b420d21a8ec74f6cf53b0bf3ed43b52/waves_gateway/gateway.py#L105-L109

However, currently it doesn't seem to be [the case anymore](https://github.com/PyWaves/PyWaves/blob/c6d1fdfca5446704ea092092a4f9169128a6fc45/address.py#L208-L216). Now when the `address` argument is provided, the `Address.__init__` doesn't bother with generating the `publicKey` anymore. A simple fix would to change the _gateway.py_ file (above), to just pass in the privateKey part to the `Address(...)`:

```python
@Factory(GATEWAY_PYWAVES_ADDRESS, deps=[GATEWAY_WAVES_ADDRESS_SECRET, WAVES_NODE, WAVES_CHAIN], opt_deps=[WAVES_CHAIN_ID])
def gateway_pywaves_address(gateway_waves_address_secret: model.KeyPair, waves_node: str, waves_chain: str, waves_chain_id: Optional[str] = None):
    """Creates an address instance from the pywaves library that represents the Gateway waves address."""
    pywaves.setNode(waves_node, waves_chain, waves_chain_id)
    return pywaves.Address(privateKey=gateway_waves_address_secret.secret)
```

Though maybe there's also some usecase which entails running the gateway without the account private key.

**Comment by PyWaves** [comment]
Good observation. Please create a pull request.

**Comment by henning96** [comment]
Fixed in Release v1.0.4
